### PR TITLE
Added listing contacts UI

### LINKED ISSUE #4: Create list contacts UI

### PR DESCRIPTION
**Pull request by mr-jestin-roy**
Fixes #4 .
Resolved code based on review suggestions in PR #10 and #11 

Screen Shot 2022-09-13 at 4 14 57 PM

### CODE REVIEW

File: app/javascript/src/App.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
 
-import dayjs from "src/lib/dayjs"; // eslint-disable-line
+import dayjs from "lib/dayjs"; // eslint-disable-line
```
Comment (yedhink) [inline app/javascript/src/App.jsx:3]: I believe I had already pointed out the issue with this code once before. Check the Slack channel thread.

File: app/javascript/src/components/Dashboard/Contacts/Table/TableColumnData.jsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
+import React from "react";
+
+import { MenuHorizontal } from "neetoicons";
+import { Typography, Avatar, Dropdown } from "neetoui";
+
+import AvatarProfile from "../../../../../../assets/images/AvatarProfile";
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Table/TableColumnData.jsx:6]: Does this import path look fine to you even after going through https://www.bigbinary.com/books/learn-rubyonrails-book/setting-up-react-environment#aliases?

File: app/javascript/src/components/Dashboard/Contacts/index.jsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+          menuBarToggle
+          actionBlock={<Button icon="ri-add-line" label="Add Contacts" />}
+          title="All Contacts"
+          searchProps={{
+            value: searchTerm,
+            onChange: e => setSearchTerm(e.target.value),
+            placeholder: "Search Name, Email, etc..",
+          }}
+        />
+        <Table showAlert={() => handleDeleteAlert()} />
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/index.jsx:36]:
```suggestion
<Table showAlert={handleDeleteAlert} />
```

File: app/javascript/src/components/Dashboard/Notes/constants.js [hunk 4]
```diff
@@ -110,3 +112,22 @@ export const MenuBarBlocks = {
...
+    .nullable()
+    .required("Assigned Contact is required."),
+  tag: yup
+    .object({
+      label: yup.string().required(),
+      value: yup.string().required(),
+    })
+    .nullable()
+    .required("Tag is required."),
+});
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/constants.js:133]: So can't we assign multiple tags to a note? If not, what's the business reason for the same?

**Review by yedhink — COMMENTED** [review]
@mr-jestin-roy

**Comment by mr-jestin-roy** [comment]
@yedhink I will go through & fix these code changes before moving to the next issue.

**Pull request merged**